plugin/src/main/java/si/kamino/gradle/lokalise/LokaliseBasePlugin.kt

```diff
@@ -61,11 +60,10 @@ class LokaliseBasePlugin : Plugin<Project> {
         val retrofit = Retrofit.Builder()
             .baseUrl("https://api.lokalise.co/api2/projects/")
             .addConverterFactory(converterFactory)
-            .addCallAdapterFactory(CoroutineCallAdapterFactory())
```

Comment:
are you sure?

Reply:
The lib and `CoroutineCallAdapterFactory` is deprecated as of retrofit 2.6.0 , when it got official `suspend` support.